modules/browser/BrowserCapabilities.js

```diff
@@ -266,6 +277,17 @@ export default class BrowserCapabilities extends BrowserDetection {
                     !== 'undefined');
     }
 
+    /**
+     * Checks if the browser supports getUserMedia.
+     * @returns {boolean} {@code true} if the browser supports getUserMedia.
+     */
+    supportsGetUserMedia() {
+        return typeof navigator.getUserMedia !== 'undefined'
+            || (typeof navigator.mediaDevices !== 'undefined'
```

Comment:
You can directly add this check `typeof navigator.mediaDevices !== 'undefined' && typeof navigator.mediaDevices.getUserMedia !== 'undefined'` to `isWebKitBased()` instead of declaring a function since this function won't be used anywhere else. Sorry if my previous comment was confusing.

Reply:
Got it 👍